keyrings/alt/file.py

```diff
@@ -37,7 +39,8 @@ class Encrypted(object):
     """
     PyCrypto-backed Encryption support
     """
-
+    scheme = 'PyCrypto [PBKDF2] AES256.CFB'
```

Comment:
Might be nice if the scheme were implementation agnostic - such that it only specifies the algorithm, and implementations could be swapped out. Is there a standard for doing that? When I encrypt something using https://www.infoencrypt.com/, it creates a header indicating the algorithm. Perhaps that's a model to follow. Just an idea for consideration, not a blocker for this change.